packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -311,11 +326,16 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
       }
 
       await registerAddIn(manifestPath);
-
       // for Outlook, open Outlook.exe; for other Office apps, open the document
-      sideloadFile = (app === OfficeApp.Outlook)
-        ? await getOutlookExePath()
-        : await generateSideloadFile(app, manifest, document);
+      if (app == OfficeApp.Outlook) {
+        const version = await getOutlookVersion() ?? "";
+        if (version < "16.0.13709.10000") {
```

Comment:
This is a string value so I think we need to convert this to a number and  compare it to the number 16.0.13709.10000

Reply:
For version numbers, you need to compare each numeric portion separately.

Reply:
Why can't I compare them as strings directly?
The versions are in alphabetical order!

Below there's a StackOverflow item on how to compare strings:
https://stackoverflow.com/questions/10198257/comparing-2-strings-alphabetically-for-sorting-purposes#:~:text=JavaScript%20compares%20strings%20character%20by%20character%20and%20%22a%22,all%20equal,%20lets%20move%20to%20the%20next%20character.

Reply:
Version numbers do not follow alphabetic order.

"16.0.30" is greater than "16.0.130" alphabetically, but 16.0.130 is a bigger version number.

Reply:
You can do something like we do in generator office for checking the version of node:

    if (parseInt(process.version.slice(1, process.version.indexOf('.'))) % 2 == 1) {
      console.log(yosay('generator-office does not support your version of Node. Please switch to the latest LTS version of Node.'));
      this._exitProcess();
    }
We really are only interested in this part of the version: 16.0.13709

Reply:
Also you can change your local version to Production, by going to Outlook File--> Office Account--> Office Account and selecting the Office Insider button. This gives you the option to the "Change Channel" (I assume you are currently in the Dogfood Channel, but you can change it to "Microsoft Current Channel (OuterRing)" in the following dialog:

![image](https://user-images.githubusercontent.com/38081557/109327705-7ce19a80-780d-11eb-82a8-06ab2916ea11.png)
 
Be sure you check the privacy checkbox or in my experience the settings don't take effecr.

You can then go to Office Updates under Outlook File--> Office Account--> Office Account and do Update Now and this will install a Production version of Office so you will have a version >= 16.0.13709.10000

After you are done testing, you probably want to change your channel back "Dogfood Main Channel" for reapply the updates to install back to your original version

Reply:
I just updated the PR by using a package named ```semver```, that compares the versions directly. So that should be taken care of now.